dla_cnn/desi/preprocess.py

```diff
@@ -164,4 +164,6 @@ def rebin(sightline, v = 20000):
     sightline.flux = new_fx[left:right]
     sightline.error = new_var[left:right]
     
+    sightline.flux[sightline.flux<0] = 0.001
```

Comment:
This should not be required.  Please remove.

Reply:
Ok, I have discussed this with @caiz12 and understood why flux can be negative for spectrum. I have removed it. Thanks!